modules/cas_cache/utils/utils_mpool.h

```diff
@@ -6,55 +6,53 @@
 #ifndef UTILS_MPOOL_H_
 #define UTILS_MPOOL_H_
 
-#define ALLOCATOR_NAME_MAX 128
+#include <linux/types.h>
+
+#define MPOOL_ALLOCATOR_NAME_MAX 128
 
 enum {
-	cas_mpool_1,
-	cas_mpool_2,
-	cas_mpool_4,
-	cas_mpool_8,
-	cas_mpool_16,
-	cas_mpool_32,
-	cas_mpool_64,
-	cas_mpool_128,
+	env_mpool_1,
+	env_mpool_2,
+	env_mpool_4,
+	env_mpool_8,
+	env_mpool_16,
+	env_mpool_32,
+	env_mpool_64,
+	env_mpool_128,
 
-	cas_mpool_max
+	env_mpool_max
 };
 
-struct cas_mpool {
-	uint32_t item_size;
-		/*!< Size of specific item of memory pool */
-
-	uint32_t hdr_size;
-		/*!< Header size before items */
+struct env_mpool;
 
-	env_allocator *allocator[cas_mpool_max];
-		/*!< OS handle to memory pool */
-
-	int flags;
-		/*!< Allocation flags */
+struct env_mpool_init_limit {
+	uint32_t size;
+	int rpool_limit;
```

Comment:
maybe change the type to mpool item size enumeration (the one right above this struct)

Reply:
changed to simple array of limits